### PR TITLE
Show consistent zero states

### PR DESCRIPTION
**Pull request by cnwalker**
I couldn't find any zero states other than storageClass so far that are inconsistent with the paradigm @floreks laid out in #2121. Any others I should take a look at?

### CODE REVIEW

**Comment by maciaszczykm** [comment]
Check https://github.com/kubernetes/dashboard/issues/2121#issuecomment-316943970. Did you check replica sets, replication controllers etc?

**Comment by cnwalker** [comment]
Ah, I don't think I fully understood until just now. Will update the PR shortly.

**Comment by cnwalker** [comment]
@maciaszczykm I think I found many more cases.

**Comment by maciaszczykm** [comment]
I will review it tomorrow or the day after.

**Comment by maciaszczykm** [comment]
CI failed due to lint error.

**Comment by maciaszczykm** [comment]
Tested it and it looks really well, a lot more consistent :)

File: src/app/frontend/thirdpartyresource/list/list.html [hunk 1]
```diff
@@ -14,11 +14,19 @@
...
+<kd-content-card ng-if="$ctrl.shouldShowZeroState()">
   <kd-content>
-    <kd-third-party-resource-card-list third-party-resource-list="$ctrl.thirdPartyResourceList"
-                                       third-party-resource-list-resource="$ctrl.thirdPartyResourceListResource"
-                                       with-statuses="false">
-    </kd-third-party-resource-card-list>
+    <kd-zero-state></kd-zero-state>
   </kd-content>
-</kd-content-card>
+</kd-content-card>
```
Comment (maciaszczykm) [inline src/app/frontend/thirdpartyresource/list/list.html]: Missing new line.

**Comment by cnwalker** [comment]
Looks like it was a lint issue and an issue with global scope of the ctrl variable in the `controller_test` modules. Ready for another review @maciaszczykm!

**Comment by maciaszczykm** [comment]
@cnwalker Thanks :)

**Pull request merged**